stream/src/public_key/benches/connection.rs

```diff
@@ -0,0 +1,67 @@
+use chacha20poly1305::{ChaCha20Poly1305, KeyInit};
+use commonware_runtime::mocks;
+use commonware_stream::{public_key::Connection, Receiver, Sender};
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use futures::executor::block_on;
+
+fn benchmark_sender(c: &mut Criterion) {
+    let msg = b"hello";
+    let max_message_size = 1024 * 1024; // 1MB
+
+    c.bench_function(
+        &format!(
+            "max_message_size_len={} msg_len={}",
```

Comment:
criterion works best if you ensure each benchmark has a unique name. I'd recommend doing something like `sender...` and `receiver...`